CHANGELOG.md

```diff
@@ -32,6 +32,17 @@ Table of Contents
 ### Improvements
 
 
+--------------------------------------------------------------------------------
+
+[1.9.0]
```

Comment:
This goes first into unreleased, and only with the release, we add a version

Reply:
changed